src/System.Text.Http.Parser/HttpVersion.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace System.Text.Http.Parser
+{
+    public enum HttpVersion
+    {
+        Unknown = -1,
+        Http10 = 0,
+        Http11 = 1
+    }
```

Comment:
Similarly good at some point to rationalize this with https://github.com/dotnet/corefx/blob/master/src/System.Net.Primitives/src/System/Net/HttpVersion.cs